lib/src/package_analysis/shapes_ext.dart

```diff
@@ -60,3 +60,19 @@ extension LibraryShapeExt on LibraryShape {
         exportedClasses: exportedClasses ?? this.exportedClasses,
       );
 }
+
+extension PackageShapeExt on PackageShape {
+  /// Returns a list of the names of all the top-level functions which are defined in this package.
+  List<String> get getFunctions {
+    return functions.map((function) => function.name).toList();
+  }
+
+  /// Returns a list of the names of all the class methods which are defined in this package.
+  List<String> get getMethods {
```

Comment:
```suggestion
  List<String> get allMethodNames {
```

But consider doing `bool hasSomeClassWithMethod(String name)`